src/main/java/frc/robot/subsystems/Revolver.java

```diff
@@ -19,18 +19,18 @@
 
 public class Revolver extends SubsystemBase {
 
-  private final CANSparkMax m_revolver;
+  private final TalonFX m_revolver;
```

Comment:
Change this to `WPI_TalonFX` to allow access to WPILib `SpeedController` methods

Reply:
Resolved in commit 4b623d01